enterprise/server/backends/userdb/userdb.go

```diff
@@ -34,9 +34,9 @@ const (
 )
 
 var (
-	addUserToDomainGroup = flag.Bool("app.add_user_to_domain_group", false, "Cloud-Only")
-	createGroupPerUser   = flag.Bool("app.create_group_per_user", false, "Cloud-Only")
-	noDefaultUserGroup   = flag.Bool("app.no_default_user_group", false, "Cloud-Only")
+	addUserToDomainGroup = flag.Bool("app.add_user_to_domain_group", true, "Cloud-Only")
+	createGroupPerUser   = flag.Bool("app.create_group_per_user", true, "Cloud-Only")
+	noDefaultUserGroup   = flag.Bool("app.no_default_user_group", true, "Cloud-Only")
```

Comment:
I don't think we want these to be the defaults. They're only true for BuildBuddy Cloud.